### PR TITLE
Add TemplateBooleanAnd (annotated)

### LINKED ISSUE #131: [PE-D] User Guide lacks standardized formatting
The presence of a table to summarize keywords is great but other than code, the rest are only used 3 times or irrelevantly. 

Irrelevant use of bolding. Provides no significant information to user.

For code, the bucket is too broad and confuses users as to what they should see and what they should type into the CLI. It is preferable to use screenshots instead for what they should see and that should reduce your issues by quite a fair bit.

Unclear which should I actually put into my CLI or should I see. Surprise! It's none of the two. Only the last line follows the standardized guide. This is repeated consistently across the UG.

-------------
Labels: `severity.High` `type.DocumentationBug`
original: brandonywl/ped#7

### LINKED ISSUE #154: User Guide Issues
- [x] #124 
    - **Circuit `add` command**: 
        - Update `add` command that `v` cannot be added, only set.
- [x] #125 
    - **Circuit Action Command**: 
        - Add to Circuit Action Command preface that templates are only simplified circuits with GUI-constraint.
- [x] #126 
    - **ALL**: Change all commands from being under "Format:" to the section names themselves
- [x]  #127 
    - **Circuit `add`, `set`, `calc` commands**: 
        - Add warning that components not part of the circuit cannot be set to `add`, `set`, `calc` commands
- [x] #128 
    - **Circuit `calc` command**: 
        - Fix example usage of `calc` 
- [x] #129, #135, #137, #142, #143, #144 
    - **Tutorial Circuit/Boolean commands**: 
        - Update instructions on how to use `tutorial`, explain how it will remain in a loop
- [x] #130 
    - **Command Preface**: 
        - Add to Command Preface that Command Summary is at the bottom of the UG.
- [x] #131 
    - **ALL**: 
        - Differentiate user input and commands using the `code` formatting. From #126 above section names could be the format, with only diagrams/ very clear separation for the **_Expected Output_**. Use **bold** for very special keywords.
- [x] #132
    - **ALL**: 
        - Remove mentions of `help` command everywhere, including Quick Start.
- [x] #133 
    - **ALL**: 
        - Numbering of all sections.
- [x] #136, #145 
    - **Circuit `add` command**: 
        - Update `add` command Expected Output.
- [x] #139 
    - **Summary command**: 
        - Add diagram of `summary` command Expected Output. Possibly add tables of parameters for every command with parameters.
- [x] #141 
    - **Command Preface**: 
        - Add that extra arguments will not be parsed by the application to Command Preface.
- [x] #146 
    - **Boolean `set` command**: 
        - Warning for setting of existing gate under `set` boolean command. Also can update `add` to make the warning of depth of logic gate clearer.
- [x]  this
    - **ALL**
        - Double check all links.
    - **Logging**
        - Add that there is a logging file created - add relevant DevGuide section for more info.
    - **Boolean `set` command**
        - Add that too large an integer will not set the input, or restrict to 0 or 1 only.

### PR DESCRIPTION
**Pull request by dorianfong98**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -333,20 +333,8 @@ This command creates a boolean template. Similar to the [circuit template](#6-ci
...
-      OUT
-       |
-   B       C
-
-OUT = B AND C
-B = ?
-C = ?
+The diagram below shows an example when using the `template and` command.
+![Simplified](diagrams/TemplateBooleanAnd.png)
 ```
```
Comment (hughjazzman) [inline docs/UserGuide.md]: Broken. Also why not the same format as the other images? The colours and all.
```suggestion

![Simplified](diagrams/TemplateBooleanAnd.png)

```

**Review by hughjazzman — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (dorianfong98) [reply docs/UserGuide.md]: Changed, let me know if it's okay

Comment (hughjazzman) [reply docs/UserGuide.md]: Yep image is ok, but formatting still broken. Commit the suggestion ⬆️

File: docs/UserGuide.md [hunk 2]
```diff
@@ -333,20 +333,8 @@ This command creates a boolean template. Similar to the [circuit template](#6-ci
...
-
-```
-      OUT
-       |
-   B       C
-
-OUT = B AND C
-B = ?
-C = ?
+The diagram below shows an example when using the `template and` command.
```
Comment (hughjazzman) [inline docs/UserGuide.md]: For consistency.
```suggestion
The image below shows an example when using the `template and` command.
```

**Review by hughjazzman — APPROVED** [review]
LGTM!

**Pull request merged**